packages/hierarchy-browser/src/CommunityCard/CommunityCard.tsx

```diff
@@ -26,35 +28,45 @@ export interface ICommunityCardProps {
 	level: number
 	hierachyDataProvider: HierarchyDataProvider
 	incrementLevel?: boolean // adjust from 0 to 1 based indexing on levels if needed
-	isOpen?: boolean
+	neighborsLoaded: boolean
+	neighborCallback?: ILoadNeighborCommunities
+	settings: ISettingState
 }
 
 const ENTITY_LOADER_MSG = 'Fetching entity data...'
 
 export const CommunityCard: React.FC<ICommunityCardProps> = memo(
 	function CommunityCard({
-		isOpen: isOpenProp,
 		community,
 		maxSize,
 		maxLevel,
 		level,
 		incrementLevel,
+		neighborsLoaded,
 		hierachyDataProvider,
+		neighborCallback,
+		settings,
 	}: ICommunityCardProps) {
 		const [dataProvider] = useState<CommunityDataProvider | undefined>(
 			() => new CommunityDataProvider(community, hierachyDataProvider, level),
 		)
+		const {
+			isOpen: isOpenProp,
```

Comment:
If this is moving into the settings prop, this will have major semver impact on this package.

Reply:
I updated to major version change for hierarchy-browser and decline for workspace but can change it if that doesn't work.